tools/integration_tests/write_large_files/seq_write_large_file_test.go

```diff
@@ -23,29 +23,36 @@ import (
 	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/setup"
 )
 
-const FiveHundredMB = 500 * 1024 * 1024
+const OneMB = 1024 * 1024
+const FiveHundredMB = 500 * OneMB
 const FiveHundredMBFile = "fiveHundredMBFile.txt"
-const ChunkSize = 20 * 1024 * 1024
+const ChunkSize = 20 * OneMB
+const DirForSeqWrite = "dirForSeqWrite"
+const FiveHundredMBFileForSeqWriteInLocalSystem = "fiveHundredMBFileForSeqWriteInLocalSystem"
 
 func TestWriteLargeFileSequentially(t *testing.T) {
-	// Clean the mountedDirectory before running test.
-	setup.CleanMntDir()
-
-	filePath := path.Join(setup.MntDir(), FiveHundredMBFile)
+	seqWriteDir := path.Join(setup.MntDir(), DirForSeqWrite)
+	err := os.Mkdir(seqWriteDir, setup.FilePermission_0600)
+	if err != nil {
+		t.Errorf("Error in creating directory:%v", err)
+	}
+	filePath := path.Join(seqWriteDir, FiveHundredMBFile)
 
 	// Sequentially read the data from file.
-	err := operations.WriteFileSequentially(filePath, FiveHundredMB, ChunkSize)
+	err = operations.WriteFileSequentially(filePath, FiveHundredMB, ChunkSize)
 	if err != nil {
 		t.Errorf("Error in writing file: %v", err)
 	}
 
-	// Check if 500MB data written in the file.
-	fStat, err := os.Stat(filePath)
+	// Download the file from a bucket in which we write the content and compare with
+	// the file content we wrote in mntDir.
+	filePathInGcsBucket := path.Join(setup.TestBucket(), DirForSeqWrite, FiveHundredMBFile)
+	localFilePath := path.Join(os.Getenv("HOME"), FiveHundredMBFileForSeqWriteInLocalSystem)
+	err = compareFileFromGCSBucketAndMntDir(filePathInGcsBucket, filePath, localFilePath)
 	if err != nil {
-		t.Errorf("Error in stating file:%v", err)
+		t.Errorf("Error:%v", err)
 	}
 
-	if fStat.Size() != FiveHundredMB {
-		t.Errorf("Expecred file size %v found %d", FiveHundredMB, fStat.Size())
-	}
+	// Clean up.
+	operations.RemoveDir(seqWriteDir)
```

Comment:
Move this as a deferred call right after this directory is successfully created to avoid unintentionally failing to delete, in case of other failures.